index.html

```diff
@@ -0,0 +1,161 @@
+        <div class="horizontal-lines">
+            <hr>
+        </div>
+        <div class="sec">
+            <div class="main-section">
+                <div class="exibition">
+                    <img src="assets/images/roulette.png" alt="">
+                    <h3>Leacture</h3>
+                    <p>December 1st kicks off AIDS Awareness Month with World AIDS Day. Leading up to the month of December was the 40th Anniversary</p>
+                </div>
+                <div class="exibition">
+                    <img src="assets/images/whiteboard.png" alt="">
+                    <h3>CC Exhibition</h3>
+                    <p>With HIV related products and services on display in interactive 3D booths; chat functions enabling real-time dialogue with exhibitors</p>
+                </div>
+                <div class="exibition">
+                    <img src="assets/images/chat.png" class="forum1" alt="">
+                    <h3>Forum</h3>
+                    <p>The Forum HIV Prevention Trial Design Project convenes a series of public meetings.</p>
+                </div>
+                <div class="workshop">
+                    <img src="assets/images/teacher.png" alt="">
+                    <h3>Workshop</h3>
+                    <p>AIDS 2018 workshops are grouped into three areas: community, leadership and science.</p>
+                </div>
+                <div class="workshop">
+                    <div class="redlogo-container">
+                        <img src="assets/images/download.png" class="red-logo" alt="">
+                    </div>
+                    <h3>CC Ignite</h3>
+                    <p>AIDS 2018 workshops are grouped into three areas: community, leadership and science.</p>
+                </div>
+                <div class="global">
+                    <h4 class="bar">Join HIV Global Summit 2022</h4>
+                </div>
+            </div>
+        </div>
+        <div   class="program">
+            <a href="">SEE THE WHOLE PROGRAM</a>
+        </div>
+</section>
+<div id="speaker-list"></div>
+<h2 class="speaker-list">Featured Speakers</h2>
+<div class="center-hor">
+    <div class="horizontal-line"></div>
+</div>
+<div id="speaker-lists"></div>
+<footer>
+    <div class="footer">
+        <h2>Partner</h2>
+        <div class="sub-section">
+            <div class="footer-section">
+                <ul class="lists">
+                    <li>Have Questions ?</li>
+                    <li>Privacy</li>
+                    <li>Connect with CDC</li>
+                    <li class="link-privacy">Policies</li>
+                    <li>Funding</li>
+                </ul>
+            </div>
+        </div>
+    </div>
+    <div class="footer-1">
+        <h3>Uganda. Department of Health & Human Services</h3>
+        <ul class="u-list">
+            <li>Provided in collaboration with NIH’s Office of AIDS Research.</li>
+            <li>Freedom of Information Act</li>
+        </ul>
+    </div>
```

Comment:
- [ ] As requested by the previous reviewer, Please fix the footer content, Reduce the heading size make sure it matches with the template.

### Your design

![image](https://user-images.githubusercontent.com/83803414/169514876-7a7444ee-8d6b-4c28-83d7-4c142a0d57fe.png)

### Behance design

![image](https://user-images.githubusercontent.com/83803414/169514911-15e6e89b-80d7-4e41-aa34-87942156ea5c.png)

Reply:
- [ ]  Kindy fix the design in your footer section you need to include a logo and text, currently, you have text only.